### DISCUSSION TITLE
I don't want that when I click on a button inside a slider, it brings that slider to the first position

### DISCUSSION THREAD

**Discussion by LukaLiuzzi** [nolimits4web/swiper#5448] [Q&A]
### When I click on a button that I have inside my slider, instead of executing what the button should do, what it does is take that slider to position 1 of the carousel. what I want is that this does not happen since it does not allow the action of the button to be executed, which is to open an overlay. I leave a video so that it is better understood.

https://sendvid.com/rkza0bku

**Comment by gabiito** [comment]
I have the same issue @LukaLiuzzi is mentioning.

For now, the way around this issue that I found is using an `<a>` instead of a  `<button>` and then it works like it should.
